.gitignore

```
@@ -13,3 +13,4 @@
 .externalNativeBuild
 .cxx
 local.properties
+/buildsrc/build
```

```suggestion
buildSrc/build
```
Should it be `buildSrc` instead?